### PR TITLE
Start message for node

### PR DESCRIPTION
**Pull request by vimukthi-git**
- This fixes duplicate protobuf errors when node starts
- Adds a "beta software" message
- Adds a ascii art

### CODE REVIEW

File: protobufs/invoice/service.proto [hunk 1]
```diff
@@ -1,11 +1,11 @@
 syntax = "proto3";
 
-package invoice;
+package inv;
 
-option go_package = "invoicepb";
+option go_package = "invpb";
```
Comment (mikiquantum) [inline protobufs/invoice/service.proto]: can we add the key `api` or `client` somewhere instead?

File: cmd/centrifuge/root.go [hunk 2]
```diff
@@ -39,6 +41,12 @@ func Execute() {
 	}
 }
 
+func printStartMessage() {
+	c := figure.NewFigure("Centrifuge OS", "", true)
```
Comment (mikiquantum) [inline cmd/centrifuge/root.go]: Do you have a screenshot you can attach of how it would look like?

File: cmd/centrifuge/root.go [hunk 3]
```diff
@@ -39,6 +41,12 @@ func Execute() {
...
+	c.Print()
+	fmt.Println("Centrifuge is beta software !!!")
```
Comment (mikiquantum) [inline cmd/centrifuge/root.go]: Do we need to add more information about what this beta version means?

File: protobufs/invoice/service.proto (hunk 1 above)
Comment (vimukthi-git) [reply protobufs/invoice/service.proto]: that was the first thing I tried, its just too long.. and we are removing these soon anyway

File: cmd/centrifuge/root.go (hunk 2 above)
Comment (vimukthi-git) [reply cmd/centrifuge/root.go]: Screenshot 2019-04-10 at 11 30 17

File: cmd/centrifuge/root.go (hunk 3 above)
Comment (vimukthi-git) [reply cmd/centrifuge/root.go]: I also thought of this, but for the console its too much information. I think we can explain that in the disclaimer on develop website.

**Pull request merged**